src/main/java/com/dokany/java/DokanyOperationsProxy.java

```diff
@@ -121,8 +126,8 @@
 		public long callback(WString rawPath, WinBase.SECURITY_ATTRIBUTES securityContext, int rawDesiredAccess, int rawFileAttributes, int rawShareAccess, int rawCreateDisposition, int rawCreateOptions, DokanyFileInfo dokanyFileInfo) {
 			try {
 				//hack to uniquely identify the dokan threads
-				if (Thread.currentThread().getName().equals(DEFAULT_THREAD_NAME)) {
-					Thread.currentThread().setName("dokan-" + threadCounter.getAndIncrement());
+				if (Thread.currentThread().getName().equals(nativeThreadNamePrefix)) {
+					Thread.currentThread().setName(nativeThreadNamePrefix + threadCounter.getAndIncrement());
```

Comment:
Where is this Thread created? Can we change the ThreadFactory in order to set the name a priori?

Reply:
> Where is this Thread created? 

Those threads are the native callback threads of Dokany. I removed this hack and implemented our own CallbackThreadInitializer (see f02686b)